st-embroider/2024/2025-04-01.md

```diff
@@ -0,0 +1,67 @@
+# 2024-04-01
+
+Note Taker: Chris Manson
+Time Keeper: Chris Manson
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+- [x] Ed Faulkner (EF)
+- [x] Chris Manson (CM)
+- [ ] Aaron Chambers (AC)
+- [x] Alex (A)
+- [x] Preston Sego (PS)
+- [x] Simon Ihmig (SI)
+- [ ] Katie Gengler (KG)
+- [ ] Peter Meehan (PM)
+- [ ] Krystan HuffMenne (KH)
+- [x] Peter Wagenet (PW)
+- [ ] James Davis (JD)
+- [x] David Taylor
+- [x] Johnard
+- [x] Tommyjr
+
+## Topics
+
+
+
+### Should @ember/test-helpers, @ember/test-waiters, be put in ember-source? <@NullVoxPopuli> 
+
+- Test-helpers, specifically, relies on private APIs
+- EF: what are we being using
+- PS: we are using things like setOnError
+- EF: is the need for this because we don't have setComponent
+- I would prefer to stabilise the apisthat are being used
+- (discussion of what the actual things are)
+- EF: a lot of the examples are things that are on the way out
+- we could just wait and remove them when they're not being used any more
+
+
+### what should happen with https://github.com/emberjs/ember-test-helpers/pull/1528 <@NullVoxPopuli>
+
+- EF: looks good
+- we need to fix the conflict to continue
+- 
+
+### A new attempt at my strategy of splitting ember-cli and the blueprint system: https://github.com/ember-cli/ember-cli/pull/10671 <@mansona>
+
+- (long discussion about the strategy)
+- EF: we can do this but don't change behaviour i.e. don't resolve from cwd
```

Comment:
- most of what ember-try's complexity is around local running and trying to support monorepos, which have seemingly unlimited configuration issues